pkg/build/controller/config_controller.go

```diff
@@ -79,7 +79,7 @@ func (c *BuildConfigController) HandleBuildConfig(bc *buildapi.BuildConfig) erro
 		if kerrors.IsConflict(err) {
 			instantiateErr = fmt.Errorf("unable to instantiate Build for BuildConfig %s/%s due to a conflicting update: %v", bc.Namespace, bc.Name, err)
 			utilruntime.HandleError(instantiateErr)
-		} else if buildgenerator.IsFatal(err) {
+		} else if buildgenerator.IsFatal(err) || kerrors.IsNotFound(err) || kerrors.IsBadRequest(err) {
```

Comment:
And the generateBuildConfig and updateBuildConfig methods.  Are all those
fatal?

On Wed, Jul 13, 2016 at 5:21 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/build/controller/config_controller.go
> <https://github.com/openshift/origin/pull/9828#discussion_r70711225>:
>
> > @@ -79,7 +79,7 @@ func (c *BuildConfigController) HandleBuildConfig(bc *buildapi.BuildConfig) erro
> >  		if kerrors.IsConflict(err) {
> >  			instantiateErr = fmt.Errorf("unable to instantiate Build for BuildConfig %s/%s due to a conflicting update: %v", bc.Namespace, bc.Name, err)
> >  			utilruntime.HandleError(instantiateErr)
> > -		} else if buildgenerator.IsFatal(err) {
> > +		} else if buildgenerator.IsFatal(err) || kerrors.IsNotFound(err) || kerrors.IsBadRequest(err) {
>
> whatever the instantiate api says is a bad request, eg:
>
> https://github.com/openshift/origin/blob/master/pkg/build/generator/generator.go#L222-L226
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9828/files/64a7ffb0efa0dc6bf954fe8e43d249281a80c7ab#r70711225>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p6V1MG5OvuArozXzYhdT5cpD0wA6ks5qVVbQgaJpZM4JL2_i>
> .
>